components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/PasswordGrantHandlerTest.java

```diff
@@ -116,6 +119,9 @@ public void init() throws Exception {
 
         // Set the static field to the mock object
         logField.set(null, mockLog);
+
+        PrivilegedCarbonContext.startTenantFlow();
```

Comment:
Can we stop the tenant flow in a tearDown method witch invokes after each method?

Reply:
done